toolkit/docs/formats/imageconfig.md

```diff
@@ -194,6 +194,29 @@ A sample PackageLists entry pointing to three files containing package lists:
 ],
 ```
 
+### Disabling Documentation and Locales
+
+For size constrained images it may be desirable to omit documentation and non-default locales from an image.
+
+``` json
+"DisableRpmDocs": true,
+"OverrideRpmLocales": "NONE",
+```
+
+`DisableRpmDocs` will configure the image to never install the documentation. Selecting `OverrideRpmLocales=NONE` will omit all locales from the image.
+
+These customizations are encoded into `/usr/lib/rpm/macros.d/macros.installercustomizations_*` files on the final system.
```

Comment:
I'd also exclusively mention that we're using RPM's `%_excludedocs ` macro to disable the installation of docs on the final system. Similar to how you mention `%_install_langs` for the locales below.

Reply:
```suggestion
These customizations are encoded into `/usr/lib/rpm/macros.d/macros.installercustomizations_*` files on the final system. They set values for `%_excludedocs` and `%_install_langs` respectively.
```